scout/demo/rnafusion.load_config.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+owner: cust000
```

Comment:
I'm giving this demo track a try and it loads! 🥇 

<img width="1235" alt="image" src="https://user-images.githubusercontent.com/28093618/216036277-19426670-2510-47e9-a485-acf95d062673.png">

Perhaps we don't worry about the HTML files for now, they are not created by Scout anyway!

Reply:
Agreed - I've flagged them down in Sonarclound for now, but we should alert the rna-fusion devs as well - or is that you now @fevac?

Reply:
The thing is that the nf-core/rnafusion pipeline is using other external tools to create these reports.  So I don't think the rnafusion developers will be the ones addressing this but instead the developers of those external tools. I could have a look at their repos and make an issue in those.